acsylla/_cython/cpp_cassandra.pxi

```diff
@@ -8,7 +8,11 @@ cdef extern from "cassandra.h":
     cass_true = 1
 
   ctypedef enum CassError:
-    CASS_OK
+    CASS_OK = 0
+    CASS_ERROR_LIB_INDEX_OUT_OF_BOUNDS = 16777227
```

Comment:
isn't there a better way to get processors into cython ? 
I remember years ago when using SWIG I've create variables backed with c functions on some python object for the sake of having those

Reply:
this seem like a good option: https://stackoverflow.com/a/5705865

Reply:
Good point, just removed the values and it works like a charm.